be1-go/message/messagedata/coin_post_transaction.go

```diff
@@ -52,17 +53,28 @@ type UnlockScript struct {
 	Sig    string `json:"sig"`
 }
 
+// SumOutputs computes the sum of the amounts in all outputs.
+// It may signal an overflow error
+func (transaction Transaction) SumOutputs() (acc uint53.Uint53, err error) {
```

Comment:
we tend to never use named return values:

```suggestion
func (transaction Transaction) SumOutputs() (uint53.Uint53, error) {
  var acc uint53.Uint53
  var err error
```